dcp-prototype/consumer/README.md

```diff
@@ -0,0 +1,30 @@
+# Consumer Prototype
```

Comment:
As per our offline conversation, please avoid using consumer/contributor and instead use the module name itself.